File: section4/movieGenre.py
class Movie():
    def __init__(self):
        self.uuid = ''
        self.title = ''
        self.year = ''
        self.genres = Genre()
        self.related = []


    def addRelatedMovie(self,m):
        if(isinstance(m,Movie)):
            self.related.append(m) 
            return True
        return False

    def setGenre(self,g):
        if(isinstance(g,Genre)):
            if(g.addMovie(self)):
                return True
            return False
        return False 

            
                




class Genre():
    def __init__(self):
        self.name = ''
        self.movies = []
        
    def addMovie(self, m):
        if(isinstance(m,Movie)):
            self.movies.append(m)
            return True
        return False

File: section4/movieGenre.js
class Movie{
constructor (){
    this.uuid = ""
    this.title = ""
    this.year = ""
    this.genre = ""
    this.related = []
}


addRelatedMovie(m){
    if(m instanceof Movie){ 
        this.related.push(m)
        return true
    }
    false
     }

     setGenre(g){
     if(g instanceof Genre)
     if(g.addMovie(this))
        return True
      return False
    return False
 } 

}


 

 class Genre{
   constructor (){
       this.name = ""
       this.movies = []
   }     
    addMovie(m){
        if(m instanceof Movie)
         this.movie.push(m)
           return true
        return false
    }
       
 }



 
